.pre-commit-config.yaml

```
@@ -12,6 +12,12 @@ repos:
     types: [file]
     exclude: '^(testing/btest/(Baseline|plugins|spicy|scripts)/.*|testing/builtin-plugins/.*)$'
 
+  - id: btest-at-test-commented
+    name: Check that all @TEST- lines are commented out
+    entry: '^\s*@TEST-'
+    language: pygrep
+    files: '^testing/btest/.*$'
```

Nit: It looks like we do not need to allow spaces before the BTest command, and can also run on all files, e.g.,

```suggestion
  - id: btest-command-commented
    name: Check that all BTest command lines are commented out
    entry: '^@TEST-'
    language: pygrep
```